theano/sandbox/cuda/blas.py

```diff
@@ -417,13 +419,14 @@ def __hash__(self):
         return hash(type(self)) ^ hash(self.inplace)
 
     def __setstate__(self, dct):
-        inplace = dct.get('inplace', True)
-        if inplace:
-            self.destroy_map = {0: [0]}
-        self.inplace = inplace
+        self.__dict__.update(dct)
 
-    def __getstate__(self):
-        return dict(inplace=self.inplace)
+        # Correctly reload older pickles where _op_use_c_code and
+        # destroy_map were not saved
+        if '_op_use_c_code' not in self.__dict__:
+            self._op_use_c_code = theano.config.cxx
```

Comment:
ok. But I think it is not good to create the graph and hardcode in the the value of config.cxx. I think that by default, it should be None and that mean the current value of cxx. If it is set otherwise, use that value.

Anyway, that isn't a current concern. Continue as we do for consistency for now.